### PR TITLE
FCCEUD-18: Describe module metadata

### PR DESCRIPTION
**Pull request by theashiot**
* Create module "Module metadata in Pantheon"

### CODE REVIEW

**Comment by theashiot** [comment]
@redhataccess/pantheon-developers Please review for technical completeness and accuracy

**Comment by benradey** [comment]
Thumbs up from a tech standpoint!

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[id="module-metadata-in-pantheon_{context}"]
+= Module metadata in {ProductShortName}
+
+Each module in {ProductShortName} has associated metadata for search engine optimization (SEO). SEO improves the findability of modules.  
+
+There are two types of module-metadata in {ProductShortName}:
+
+* Intrinsic:
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:8]: A descrition list makes more sense to me here:

```adoc
Intrinsic:: Metadata that is automatically inferred from the module.
```

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
++
+The metadata that is inferred from the module automatically.
++
+Example of intrinsic metadata include the following:
+
+** Module name
+** Module description
+
+* Extrinsic:
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:17]: Likewise, I'd use a description list here. :)

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
++
+The metadata that is applied manually before publishing.
++
+Examples of extrinsic metadata include the following:
+
+** Product name - the product and the product-version that the module should be published for.
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:23]: The hyphen is unnecessary in "product version".

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+[id="module-metadata-in-pantheon_{context}"]
+= Module metadata in {ProductShortName}
+
+Each module in {ProductShortName} has associated metadata for search engine optimization (SEO). SEO improves the findability of modules.  
+
+There are two types of module-metadata in {ProductShortName}:
```
Comment (robin-owen) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:6]: Unnecessary hyphen in "module metadata".

Comment (robin-owen) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:6]: Also, should we explicitly say "two types", or should we just preface the list with a generic statement?

```adoc
The types of module metadata in {ProductShortName} are as follows:
```

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc (hunk 1 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:8]: Agree, this looks to be the better option. Changed.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc (hunk 2 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:17]: changed.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc (hunk 3 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:23]: removed hyphen

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc (hunk 4 above)
Comment (theashiot) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc:6]: removed hyphen. Agree with not saying two types explicitly, changed as suggested.

**Comment by theashiot** [comment]
Thanks for the review @robin-owen! I've incorporated the changes. Take a look?

best,
ashwin

**Comment by theashiot** [comment]
@redhataccess/pantheon-developers Please merge.

**Pull request merged**